google-cloud-storage/src/main/java/com/google/cloud/storage/DefaultStorageRetryStrategy.java

```diff
@@ -145,7 +145,7 @@ private static final class EmptyJsonParsingExceptionInterceptor implements BaseI
     public RetryResult beforeEval(Exception exception) {
       if (exception instanceof IllegalArgumentException) {
         IllegalArgumentException illegalArgumentException = (IllegalArgumentException) exception;
-        if (illegalArgumentException.getMessage().equals("no JSON input found")) {
+        if ("no JSON input found".equals(illegalArgumentException.getMessage())) {
```

Comment:
Move the constant to the left side to avoid NullPointerException if there is ever an IllegalArgumentException which doesn't have a message.